index.html

```diff
@@ -4,11 +4,1380 @@
+                  </div>
+                </div>
+              </div>
+              <div class="col-sm-12 further">
+                <div class="row">
+                  <div class="col-sm-2 col-md-6 col-lg-4">
+                    <a href="#"><img class="featured-img"
+                        src="https://d.newsweek.com/en/full/1599606/kyrie-irving-brooklyn-nets.jpg?w=95&h=95&f=a18c6bbb81d1737ad2d8995e3cf98f84"
+                        alt=""></a>
+                  </div>
+                  <div class="col-sm-10 col-md-6 col-lg-8">
+                    <a href="#" class="more-info-link">Kyrie Irving Urges Players to Boycott NBA Resumption in
+                      Orlando</a>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </div>
+
+    <!-- SUBSCRIBE -->
+    <div class="window-row sub-newsletter">
+      <div class="flex-lg row">
+        <div class="col-lg-7 col-xl-8 flex-lg fd-c">
+          <div class="subscribe-row">
+            <h5>SUBSCRIBE</h5>
+            <div class="subscribe-box">
+              <div class="sub-img-container">
+                <div class="img1-container"><img
+                    src="https://d.newsweek.com/en/full/1596252/cover-june-12-2020.jpg?w=170&q=95&f=fdda5cadbe8011b01a2d540d06703b6e"
+                    alt=""></div>
+                <div class="img2-container"><img
+                    src="https://d.newsweek.com/en/full/1597761/cover-june-19-26-2020.jpg?w=170&q=95&f=5fb0215ca8fd65a8cfe24229880661d4"
+                    alt=""></div>
+              </div>
+              <div class="sub-text">
+                <h3 class="sub-h3">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h3>
+                <p>All Access + Weekly Delivery. <br>
+                  Choose A Membership That's Perfect for You!</p>
+                <button class="subscribe-btn-body header-btn" type="button">SUBSCRIBE <i class="fa fa-chevron-right"
+                    aria-hidden="true"></i></button>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-5 col-xl-4 flex-lg fd-c">
+          <div class="newsletter-row">
+            <h5>NEWSLETTER</h5>
+            <div class="newsletter-box">
+              <div class="news-img-container">
+              </div>
+              <div class="news-text">
+                <h3 class="sub-h3">The Starting 5</h3>
+                <p>See why nearly a quarter of a million subscribers begin their day with the Starting 5.</p>
+                <input type="text" placeholder="Email address">
+                <button class="subscribe-btn-body sub-btn" type="button">FREE SIGN UP <i class="fa fa-chevron-right"
+                    aria-hidden="true"></i></button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
To make your code more semantic i would encourage you to use the section tag for the different parts of your web page